Formula/zork.rb

```diff
@@ -0,0 +1,21 @@
+class Zork < Formula
+  desc "The Dugeon modified from FORTRAN to C"
+  homepage "https://github.com/devshane/zork"
+  url "https://github.com/devshane/zork/archive/v1.0.0.tar.gz"
+  version "1.0.0"
+  sha256 "ceedb436cf552bd8ca39abd3e85bae986c3f5df6f31c1bf2806d1b4c8509da67"
+  head "https://github.com/devshane/zork.git"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "make", "LIBDIR=#{lib}", "BINDIR=#{bin}"
+    bin.install "zork"
+    lib.install "dtextc.dat"
+    man6.install "dungeon.6"
```

Comment:
Oh sweet, yeah, that totally makes sense! Updated to use make install now and verified that the following works:

```
❯ brew install --build-from-source zork
==> Using the sandbox
==> Downloading https://github.com/devshane/zork/archive/v1.0.0.tar.gz
Already downloaded: /Users/ghavil/Library/Caches/Homebrew/zork-1.0.0.tar.gz
==> make LIBDIR=/opt/brew/Cellar/zork/1.0.0/lib BINDIR=/opt/brew/Cellar/zork/1.0.0/bin
==> make install LIBDIR=/opt/brew/Cellar/zork/1.0.0/lib BINDIR=/opt/brew/Cellar/zork/1.0.0/bin
🍺  /opt/brew/Cellar/zork/1.0.0: 8 files, 257.3KB, built in 3 seconds
```
```
❯ zork
Welcome to Dungeon.			This version created 11-MAR-91.
You are in an open field west of a big white house with a boarded
front door.
There is a small mailbox here.
>
```